services/web/client/source/resource/qxapp/project-v0.0.1.json

```diff
@@ -0,0 +1,209 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "simcore project",
+  "description": "Description of a simcore project",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "name",
+    "description",
+    "notes",
+    "owner",
+    "collaborators",
+    "creationDate",
+    "lastChangeDate",
+    "workbench"
+  ],
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "project name",
+      "examples": [
+        "Temporal Distortion Simulator"
+      ]
+    },
+    "description": {
+      "type": "string",
+      "description": "longer one-line description about the project",
+      "examples": [
+        "Dabbling in temporal transitions ..."
+      ]
+    },
+    "notes": {
+      "type": "string",
+      "description": "longer project description. using common mark",
+      "examples": [
+        "# title\nSome Text `with` common mark markup"
+      ]
+    },
+    "owner": {
+      "type": "string",
+      "description": "user uuid"
+    },
+    "collaborators": {
+      "type": "object",
+      "description": "UUIDs of the users/groups who should get access to this",
+      "patternProperties": {
+        "^\\S+$": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "read",
+              "write"
+            ]
+          }
+        }
+      }
+    },
+    "creationDate": {
+      "type": "string",
+      "description": "project creation date",
+      "pattern": "\\d{4}-(12|11|10|0?[1-9])-(31|30|[0-2]?\\d)T(2[0-3]|1\\d|0?[1-9])(:(\\d|[0-5]\\d)){2}Z",
+      "examples": [
+        "2018-07-01T11:13:43Z"
+      ]
+    },
+    "lastChangeDate": {
+      "type": "string",
+      "description": "last save date",
+      "pattern": "\\d{4}-(12|11|10|0?[1-9])-(31|30|[0-2]?\\d)T(2[0-3]|1\\d|0?[1-9])(:(\\d|[0-5]\\d)){2}Z",
+      "examples": [
+        "2018-07-01T11:13:43Z"
+      ]
+    },
+    "workbench": {
+      "type": "object",
+      "patternProperties": {
+        "^\\S+$": {
+          "type": "object",
+          "additionalProperties": false,
+          "required": [
+            "key",
+            "version",
+            "position"
+          ],
+          "properties": {
+            "key": {
+              "type": "string",
+              "description": "distinctive name for the node based on the docker registry path",
+              "pattern": "^(service)/(computational|dynamic)(/[^\\s/]+)+$",
+              "examples": [
+                "service/computational/sleeper",
+                "service/dynamic/3dviewer"
```

Comment:
I don't really understand what this key is with regard to the workbench. Also the examples are docker images??

Reply:
yes these are the names  of the docker images ... I have discussed this with @mguidon ... maybe he can elaborate